blocks/product-list/product-list.css

```diff
@@ -0,0 +1,150 @@
+.product-list ul {
+  --card-height: auto;
+
+  list-style: none;
+  margin: 0;
+  margin-top: var(--udexSpacer40);
+  padding: 0;
+  display: grid;
+  grid-template-columns: repeat(1,1fr);
+  grid-gap: var(--udexGridGutters);
+}
+
+@media (width >= 640px) {
+  .product-list ul {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .product-list ul.product-list-length-1 {
+    grid-template-columns: repeat(1,1fr);
+  }
+}
+
+@media (width >= 980px) {
+  .product-list ul {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .product-list ul.product-list-length-1 {
+    grid-template-columns: repeat(1,1fr);
+  }
+
+  .product-list ul.product-list-length-2 {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+.product-list ul>li {
+  border: 1px solid var(--udexColorGrey2);
+  box-shadow: 0 2px 4px rgb(34 54 73 / 20%);
```

Comment:
@benpeter : Please get in touch with @ok for a design token (= CSS variable) color here